src/SharpCompress/Readers/IReader.cs

```diff
@@ -8,6 +8,7 @@ public interface IReader : IDisposable
     {
         event EventHandler<ReaderExtractionEventArgs<IEntry>> EntryExtractionBegin;
```

Comment:
Yeah, I break things all the time :)

I keep this library "beta" just because I haven't given things enough thought nor have the time to worry about API breaking on updates.  Users should expect to at least recompile to find API breakages.